### PR TITLE
Added OSS spark pi example

### PR DESCRIPTION
**Pull request by Bhargavjd**
Sanity Test:
image

Successfully pulled oss image and Spark job executed as expected 
image

### CODE REVIEW

**Review by skandtandon — APPROVED** [review]

File: Data-Engineering/Airflow/exmample_spark_pi_oss.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    "retries": 0,
+}
+
+dag = DAG(
+    "spark_pi_oss",
+    default_args=default_args,
+    schedule_interval=None,
+    tags=["ezaf", "spark", "pi"],
+    params={
+        "Spark-image-url": Param(
```
Comment (akravacyber) [inline Data-Engineering/Airflow/exmample_spark_pi_oss.py:28]: nit: can we use a snake case or camel case for name of the param? Like `spark_image_url`, to be consistent with other dags

File: Data-Engineering/Airflow/exmample_spark_pi_oss.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            "",
+            type=["null", "string"],
+            description="Provide Python-Spark image url",
+        )
+        "Spark-image-version": Param(
```
Comment (akravacyber) [inline Data-Engineering/Airflow/exmample_spark_pi_oss.py:33]: nit: same about snake case here

File: Data-Engineering/Airflow/exmample_spark_pi_oss.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+dag = DAG(
+    "spark_pi_oss",
+    default_args=default_args,
+    schedule_interval=None,
+    tags=["ezaf", "spark", "pi"],
+    params={
+        "Spark-image-url": Param(
+            "",
```
Comment (akravacyber) [inline Data-Engineering/Airflow/exmample_spark_pi_oss.py:29]: nit: can we add also default value of the parameter? Like `gcr.io/....:x.x.x`

File: Data-Engineering/Airflow/exmample_spark_pi_oss.py [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+            type=["null", "string"],
+            description="Provide Python-Spark image url",
+        )
+        "Spark-image-version": Param(
+            "",
```
Comment (akravacyber) [inline Data-Engineering/Airflow/exmample_spark_pi_oss.py:34]: nit: same here, we can add default param value

**Review by akravacyber — APPROVED** [review]
LGTM, thanks!

**Review by ask664 — APPROVED** [review]

**Pull request merged**